parser/src/grammar.lalrpop

```diff
@@ -400,6 +394,12 @@ Expr: Expr<Id> = {
 
         Expr::TypeBindings(bindings, Box::new(body))
     },
+
+    "OpenBlock" <exprs: (<SpExpr> "Semi")*> <last: SpExpr> "CloseBlock" => {
+        let mut exprs = exprs;
+        exprs.push(last);
+        Expr::Block(exprs)
+    },
```

Comment:
```
let y =
    let x = 1
    x
   2
y
```
Would look like
```
let x = 1
x
```
applied on `2` which is not what one would expect.

Perhaps there is a better way of solving this but I am not aware of it at this time